rasa_nlu/server.py

```diff
@@ -58,6 +59,8 @@ def decorated(*args, **kwargs):
 
 def create_app(config, component_builder=None):
     rasa_nlu_app = Flask(__name__)
+    if 'cors_origins' in config:
```

Comment:
I am always doing that to avoid errors in the future. I am careful in expecting keys in a dict in all programming languages.